packages/core-plugin-api/api-report.md

```diff
@@ -687,7 +687,9 @@ export enum SessionState {
 
 // @public
 export type SignInPageProps = {
+  onSignInStarted(): void;
   onSignInSuccess(identityApi: IdentityApi_2): void;
+  onSignInFailure(): void;
```

Comment:
You'll want to add these over [here](https://github.com/backstage/backstage/blob/282d70fb1a40b20e025aa26ab88687642f312cca/packages/core-components/src/layout/SignInPage/types.ts#L38-L40) instead. These are the props for the core sign-in page component and there's no need to add them here.